state-synchronizer/src/network.rs

```diff
@@ -10,12 +10,12 @@ use network::{
     error::NetworkError,
     peer_manager::{ConnectionRequestSender, PeerManagerRequestSender},
     protocols::network::{NetworkEvents, NetworkSender},
-    validator_network::network_builder::NetworkBuilder,
+    validator_network::network_builder::{NetworkBuilder, NETWORK_CHANNEL_SIZE},
     ProtocolId,
 };
 use serde::{Deserialize, Serialize};
 
-/// StateSynchronizer network messages
+/// StateSynchronizer network messages{{
```

Comment:
nit: remove extra "{{" at the end of comment

Reply:
thanks for catching that! I should really change a keyboard